metabase/metabase
# Postpone showing syncing modal on the home page (for 0.42)

Epic: https://github.com/metabase/metabase/issues/18675

The modal should only be shown when the database has been syncing for more than 30 seconds.

How to test:
- Run a fresh Metabase instance.
- Connect to `Horrorshow` database during setup. Ping me if you don't know where to get credentials for it.
- When setup has been completed, wait for 30 seconds on the home page.
- You should see this modal.

You should be able to navigate to a different page and come back to the home page as well.
